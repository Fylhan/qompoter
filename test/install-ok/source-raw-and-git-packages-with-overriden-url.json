{
	"name": "install-ok/qompoter",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"fylhan/posibrain": "v0.2",
		"cp/fylhan": "v1.0"
	},
	"require-dev": {
		"fylhan/platphorm": "dev-master"
	},
	"repositories": {
		"fylhan/posibrain": "https://github.com",
		"fylhan/platphorm": "https://github.com"
	}
}
