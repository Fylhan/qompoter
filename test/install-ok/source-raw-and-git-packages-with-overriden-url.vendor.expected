vendor
vendor/acme-cp
vendor/acme-cp/qompoter.json
vendor/acme-cp/qompoter.pri
vendor/fylhan
vendor/fylhan/qompoter.json
vendor/fylhan/qompoter.pri
vendor/fylhan2
vendor/fylhan2/qompoter.json
vendor/fylhan2/qompoter.pri
vendor/platphorm
vendor/platphorm/README.md
vendor/platphorm/composer.json
vendor/platphorm/index.php
vendor/platphorm/src
vendor/platphorm/src/Platphorm
vendor/platphorm/src/Platphorm/AppLifecycleManager.php
vendor/posibrain
vendor/posibrain/.scrutinizer.yml
vendor/posibrain/.travis.yml
vendor/posibrain/LICENSE
vendor/posibrain/README.md
vendor/posibrain/app
vendor/posibrain/app/.htaccess
vendor/posibrain/app/brains
vendor/posibrain/app/brains/no-knowledge.json
vendor/posibrain/app/brains/sammy
vendor/posibrain/app/brains/sammy-iso
vendor/posibrain/app/console
vendor/posibrain/app/index.php
vendor/posibrain/composer.json
vendor/posibrain/doc
vendor/posibrain/doc/tchatbot-example.png
vendor/posibrain/example
vendor/posibrain/example/css
vendor/posibrain/example/css/ie.css
vendor/posibrain/example/css/ie.min.css
vendor/posibrain/example/css/style.css
vendor/posibrain/example/css/style.min.css
vendor/posibrain/example/dynamic.php
vendor/posibrain/example/js
vendor/posibrain/example/js/dynamic.js
vendor/posibrain/example/js/html5.js
vendor/posibrain/example/js/html5.min.js
vendor/posibrain/example/js/jquery.min.js
vendor/posibrain/example/static.php
vendor/posibrain/php.ini
vendor/posibrain/phpunit.xml
vendor/posibrain/src
vendor/posibrain/src/Posibrain
vendor/posibrain/src/Posibrain/AnalysedRequest.php
vendor/posibrain/src/Posibrain/IPositroner.php
vendor/posibrain/src/Posibrain/ITchatBot.php
vendor/posibrain/src/Posibrain/Positron
vendor/posibrain/src/Posibrain/Positroner.php
vendor/posibrain/src/Posibrain/RestApi
vendor/posibrain/src/Posibrain/TchatBot.php
vendor/posibrain/src/Posibrain/TchatBotConfig.php
vendor/posibrain/src/Posibrain/TchatMessage.php
vendor/posibrain/src/checkSetup.php
vendor/posibrain/src/tools.php
vendor/posibrain/test
vendor/posibrain/test/Posibrain
vendor/posibrain/test/Posibrain/TchatBotConfigTest.php
vendor/qompote.pri
vendor/vendor.pri
